src/backend/distributed/utils/citus_depended_object.c

```diff
@@ -76,6 +78,27 @@ SetLocalHideCitusDependentObjectsDisabledWhenAlreadyEnabled(void)
 }
 
 
+/*
+ * SetLocalClientMinMessages sets client_min_message locally to the given value
+ * if EnableUnsupportedFeatureMessages is set to false.
+ */
+void
+SetLocalClientMinMessages(int clientMinMessageLevel)
```

Comment:
`SetLocalClientMinMessages` -> `SetLocalClientMinMessagesIfRunningPGTests` or such. Otherwise, the name is too generic to get confused